Cython/Compiler/PyrexTypes.py

```diff
@@ -1085,6 +1085,9 @@ def generate_incref_memoryviewslice(self, code, slice_cname, have_gil):
     def generate_xdecref(self, code, cname, nanny, have_gil):
         code.putln("__PYX_XDEC_MEMVIEW(&%s, %d);" % (cname, int(have_gil)))
 
+    def generate_decref(self, code, cname, nanny, have_gil):
+        self.generate_xdecref(code, cname, nanny, have_gil)
```

Comment:
This needs a comment, too. Otherwise, someone will see it and spend a lot of time trying to fix the "obvious typo", which is probably not just a typo.